yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -287,19 +330,19 @@ private object YarnClusterDriver extends Logging with Matchers {
       sc.listenerBus.waitUntilEmpty(WAIT_TIMEOUT_MILLIS)
       data should be (Set(1, 2, 3, 4))
       result = "success"
+
+      // Verify that the config archive is correctly placed in the classpath of all containers.
```

Comment:
No need to change this. There are other asserts after the `try..catch` already. And if you're worried about the result file, that's legacy stuff that I'm pretty sure we can remove, although in a separate change.